setup/default.xml

```diff
@@ -60,7 +60,7 @@
         WHERE email = :email
     </entry>
 
-    <entry key='database.selectUsersAll'>
+    <entry key='database.selectUsers'>
```

Comment:
Another thought for improvement. Probably better to implement separately. Generate SQL automatically, with option to override in the config file.